src/command_modules/azure-cli-rdbms/azure/cli/command_modules/rdbms/_help.py

```diff
@@ -189,9 +189,9 @@ def add_helps(command_group, server_type):
                 short-summary: Create a {0} database.
                 examples:
                     - name: Create database 'testdb' in the server 'testsvr' with the default parameters.
-                      text: az {1} db create -g testgroup -s testsvr -n testdb
+                      text: az {1} db create -g testgroup -s testsvr -n testdb --admin-user myadmin --admin-password <server_admin_password> --sku-name GP_Gen4_2 --version 5.7
                     - name: Create database 'testdb' in server 'testsvr' with a given character set and collation rules.
-                      text: az {1} db create -g testgroup -s testsvr -n testdb --charset {{valid_charset}} --collation {{valid_collation}}
+                      text: az {1} db create -g testgroup -s testsvr -n testdb --charset {{valid_charset}} --collation {{valid_collation}} --admin-user myadmin --admin-password <server_admin_password> --sku-name GP_Gen4_2 --version 5.7
```

Comment:
The "placeholder" syntax should be the same.  Here you have {this} mixed with \<this\>.  Use the double curly braces.